app/src/main/java/at/gammastrahlung/monopoly_app/activities/BoardActivity.java

```diff
@@ -485,11 +496,72 @@ public void moveAvatarAfterCheating() {
     @Override
     public void onForward(int value) {
         moveAvatar();
+
+        Player currentPlayer = GameData.getGameData().getCurrentPlayer();
+       int currentFieldIndex = currentPlayer.getCurrentFieldIndex();
+        int moveValue = value+currentFieldIndex;
+        followingIndex = (moveValue) % 40;
+        MonopolyClient.getMonopolyClient().sendCurrentFieldInfo(followingIndex);
```

Comment:
The server already knows the position of the player, why do you need to send the index again?